Foundation/NSMassFormatter.swift

```diff
@@ -21,28 +20,238 @@ extension MassFormatter {
+            } else {
+                return MassFormatter.largePluralSymbol[unit]!
+            }
+        } else {
+            return unitString(fromValue: value, unit: unit)
+        }
+    }
+    
+
+    
+    /// The number of pounds in 1 stone
+    private static let poundsPerStone = 14.0
+    
+    /// Maps MassFormatter.Unit enum to UnitMass class. Used for measurement conversion.
+    private static let unitMass: [Unit: UnitMass] = [.gram: .grams,
+                                                     .kilogram: .kilograms,
+                                                     .ounce: .ounces,
+                                                     .pound: .pounds,
+                                                     .stone: .stones]
+    
+    /// Maps a unit to its short symbol. Reuses strings from UnitMass.
+    private static let shortSymbol: [Unit: String] = [.gram: UnitMass.grams.symbol,
+                                                      .kilogram: UnitMass.kilograms.symbol,
+                                                      .ounce: UnitMass.ounces.symbol,
+                                                      .pound: UnitMass.pounds.symbol, // see `unitStringDisplayedAdjacent(toValue:, unit:)`
+                                                      .stone: UnitMass.stones.symbol]
+    
+    /// Maps a unit to its medium symbol. Reuses strings from UnitMass.
+    private static let mediumSymbol: [Unit: String] = [.gram: UnitMass.grams.symbol,
+                                                       .kilogram: UnitMass.kilograms.symbol,
+                                                       .ounce: UnitMass.ounces.symbol,
+                                                       .pound: UnitMass.pounds.symbol,
+                                                       .stone: UnitMass.stones.symbol]
+    
+    /// Maps a unit to its large, singular symbol.
+    private static let largeSingularSymbol: [Unit: String] = [.gram: "gram",
+                                                              .kilogram: "kilogram",
+                                                              .ounce: "ounce",
+                                                              .pound: "pound",
+                                                              .stone: "stone"]
+    
+    /// Maps a unit to its large, plural symbol.
+    private static let largePluralSymbol: [Unit: String] = [.gram: "grams",
+                                                            .kilogram: "kilograms",
+                                                            .ounce: "ounces",
+                                                            .pound: "pounds",
+                                                            .stone: "stones"]
 }
 
+internal extension Locale {
+    /// TODO: Replace calls to the below function to use Locale.usesMetricSystem
+    /// Temporary workaround due to unpopulated Locale attributes
+    /// See https://bugs.swift.org/browse/SR-3202
```

Comment:
SR-3202 is fixed by https://github.com/apple/swift-corelibs-foundation/pull/1020 - can this workaround be removed @jszumski ?